book/http_cache.rst

```diff
@@ -1031,21 +1064,36 @@ matter), Symfony uses the standard ``render`` helper to configure ESI tags:
 
     .. code-block:: jinja
 
+        {# app/Resources/views/about.html.twig #}
```

Comment:
oh I see, it's because you weren't using a sub-directory. I think we should - I think the root dir should be for base layout kind of stuff.